packages/easysearch:elasticsearch/lib/engine.js

```diff
@@ -9,195 +9,197 @@ if (Meteor.isServer) {
  * @type {ElasticSearchEngine}
  */
 EasySearch.ElasticSearch = class ElasticSearchEngine extends EasySearch.ReactiveEngine {
-  /**
-   * Constructor.
```

Comment:
oh noes, doesn't look like this isn't respecting the meteor style guide https://github.com/meteor/meteor/wiki/Meteor-Style-Guide. Can you please re-indent it so I see which are the "real" changes?